nssp/delphi_nssp/run.py

```diff
@@ -72,18 +72,21 @@ def run_module(params):
         Nested dictionary of parameters.
     """
     start_time = time.time()
-    logger = get_structured_logger(
-        __name__,
-        filename=params["common"].get("log_filename"),
-        log_exceptions=params["common"].get("log_exceptions", True),
-    )
+    issue_date = params.get("patch", {}).get("current_issue", None)
+    source_dir = params.get("patch", {}).get("source_dir", None)
```

Comment:
I feel like this should throw an error if there is no source_dir param filled out, because in pull.py line 69
```
df_ervisits = pd.read_csv(f"{source_dir}/{issue_date}.csv")
```
it'll throw an error then

Reply:
If files don't exist, I figure we'll either get an error that we raise ourselves, or an automatic error when the file isn't found, so I don't think it matters too much either way.

Reply:
The check was [here](https://github.com/cmu-delphi/covidcast-indicators/pull/2000/files#diff-8fce7ae14a0e2097dd3da98817cd6258af5a7596b2a7117339da5ce8abffc875R118-R121) in `patch.py`, so if a date doesn't have source file, we just log out that there's no file for that date, and skip it. `run.py` would not be called at all in this case.
The `custom_run` flag also take it a step further [here](https://github.com/cmu-delphi/covidcast-indicators/pull/2000/files#diff-0f52ac57508ccf1c326cb006bdbc426344d14d0dcdbc79838a67fa0f6999ff90R89-R94). So I think this is very safe.